app.css

```diff
@@ -15,6 +15,5 @@ h1 {
   position: fixed;
   top: 50%;
   left: 50%;
-  margin-top: -50px;
-  margin-left: -100px;
+  transform: translate(-50%, -50%);
```

Comment:
```
  display: flex;
  justify-content: center;
  align-items: center;
```

Reply:
this will not work in the way you write it here. Text will be in the enter horizontally but not vertically.